### PR TITLE
Introduce glossary of terms

### PR DESCRIPTION
**Pull request by joelhawksley**
This PR introduces a glossary to the design guidelines section of primer.style.

Co-authored-by: Ash Guillaume <ashygee@github.com>

### CODE REVIEW

File: content/foundations/glossary.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Includes color scales, functional colors, spacing, fonts, etc.
+
+#### Color mode
+
+The appearance mode set by the operating system (light, dark, auto).
+
+#### Scheme
+
+Name for the "theme" in the UI appears to user in Appearance settings.
```
Comment (colebemis) [inline content/foundations/glossary.mdx:15]: The way this sentence is worded is a little confusing to me. 

I think included scheme name examples could help clarify (e.g. `light, dark, dark dimmed`)

File: content/foundations/glossary.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+#### Brand theme
+
+Includes color scales, functional colors, spacing, fonts, etc.
+
+#### Color mode
+
+The appearance mode set by the operating system (light, dark, auto).
+
+#### Scheme
```
Comment (colebemis) [inline content/foundations/glossary.mdx:13]: Maybe "color scheme" would be more self-explanatory. 

```suggestion
#### Color scheme
```

**Comment by yaili** [comment]
I'm not sure if this is the best place for this, especially as the content section is going to be here too. Can we think about the plan here next week?

File: content/foundations/glossary.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: Glossary
+---
+
+#### Brand theme
+
+Includes color scales, functional colors, spacing, fonts, etc.
```
Comment (yaili) [inline content/foundations/glossary.mdx:7]: Missing what a brand theme _is_.

File: content/foundations/glossary.mdx (hunk 1 above)
Comment (yaili) [reply content/foundations/glossary.mdx:15]: Yeah this sentence is a bit confusing 😬

**Comment by yaili** [comment]
Discussed briefly with @joelhawksley, I'm going to close this for now. My suggestion would be to start the glossary as markdown in our 🔒 primer repo (or design-systems for now). **Or**, even better, these terms would be defined wherever they fit best (for example, colour-related terms in the colour docs, etc.). Happy to have more chats about it. 💖

**Pull request closed without merging**